### ISSUE TITLE
Grayed out icon in VS code, no error highlighting, "undefined: click to reload" error message

### ISSUE THREAD

**Issue by curldivergence** [rust-analyzer/rust-analyzer#10203]
Hello,

Since recently rust-analyzer stopped showing error messages in one of my workspaces; its icon becomes grayed out after loading the project and has the popup that says `undefined: click to reload` (please see the screenshot). When I click the icon, cargo check runs again, according to the status bar, but nothing really changes :(

![image](https://user-images.githubusercontent.com/1927180/132906855-f3982359-74e7-4045-89c9-557bdcf80270.png)

Also, if it helps, I noticed that when I press that icon, file `.rustc_info.json` gets updated, and one of its records says
```
"5912810005807744388":{"success":false,"status":"exit code: 1","code":1,"stdout":"","stderr":"error: `-Csplit-debuginfo` is unstable on this platform\n\n"}
```
But unfortunately, I'm not sure if it has anything to do with rust-analyzer or where that parameter comes from.
Please let me know if you need any more info :)
Thank you!

**Comment by bjorn3** [comment]
Do you have `-Csplit-debuginfo` somewhere in the `RUSTFLAGS` env var or `.cargo/config`? Does `cargo check` work for you?

**Comment by curldivergence** [comment]
Hi, I do not have that flag, and cargo check does work.
However, I think I've just figured it out - due to an unrelated reason I've just installed the official Rust extension, and an error started to appear in the taskbar (there were none before). After clicking on it the Problems VS Code window popped up, and there was actual cargo output that pointed out that one of my project's dependencies cannot be complied due to env var problem, so I can guess this was the reason why rust-analyzer failed, too :)
So I'm closing the issue, and am very sorry for bothering.
Have a great day!
